src/server/rpc/procedures/ConnectN/ConnectN.js

```diff
@@ -0,0 +1,365 @@
+// This is the ConnectN RPC. It will maintain the game board and can be queried
+// for win/tie/ongoing as well as turn
+
+'use strict';
+
+var R = require('ramda'),
+    debug = require('debug'),
+    log = debug('NetsBlox:RPCManager:ConnectN:log'),
+    trace = debug('NetsBlox:RPCManager:ConnectN:trace'),
+    Constants = require('../../../../common/Constants'),
+    info = debug('NetsBlox:RPCManager:ConnectN:info');
+
+/**
+ * ConnectNRPC - This constructor is called on the first request to an RPC
+ * from a given room
+ *
+ * @constructor
+ * @return {undefined}
+ */
+var ConnectNRPC = function() {
```

Comment:
This should be called `ConnectN`, not `ConnectNRPC` (the "RPC" portion is redundant)